packages/contracts-bedrock/src/L2/interfaces/IOptimismSuperchainERC20.sol

```diff
@@ -33,7 +36,3 @@ interface IOptimismSuperchainERC20Extension is ISuperchainERC20Extensions, ISupe
     /// @notice Returns the address of the corresponding version of this token on the remote chain.
     function remoteToken() external view returns (address);
 }
-
-/// @title IOptimismSuperchainERC20
-/// @notice Combines Solady's ERC20 interface with the OptimismSuperchainERC20Extension interface.
-interface IOptimismSuperchainERC20 is IERC20Solady, IOptimismSuperchainERC20Extension { }
```

Comment:
We should leave this interface, maybe changing `IERC20Solady` for `IERC20` OZ interface